### DISCUSSION TITLE
v0.91 released!

### DISCUSSION THREAD

**Discussion by baffler** [baffler/Transparent-Twitch-Chat-Overlay#98] [General]
## v0.91 - Changes
* TwitchLib support for points redemption
* Filter settings will let you highlight certain usernames/mods/vip
* Possible bug fix for startup crash Load() issue.
* System tray icon will always be enabled for now (to prevent no interaction with app)

## Installation and Usage
* You can download the Installer now to make it easier to install the program and it will automatically install the required dependencies (VC++ runtimes).
* "Windows protected your PC" message may appear which is the Windows Defender Smart Screen. You'll need to click "More info" and then "Run anyway". You'll only have to do this the first time you launch it.
* After launching the program for the first time, you'll need to set your twitch channel in the settings. You can click the button at the top-left to open settings, right-click the black border at top, right-click the icon in the taskbar, or use the tray icon
* You can move the application by clicking and dragging the thin black border at top, and you can resize it by clicking and dragging on any part of the border
* To hide the borders, just click the [o] button in the top-right, or right-click the icon in your taskbar

![2018-11-26_18-45-47](https://user-images.githubusercontent.com/1980600/49049166-daf52280-f1ab-11e8-94ed-62568029ea03.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/tag/v0.91'>v0.91</a>.</em>

**Comment by ImAFreakenCyborg** [comment]
OMG! You're my hero! It's fixed! Thank you so so much! this program helps me so much and I am so grateful for you creating it!
